documentation/api/query/v4/index.markdown

```diff
@@ -55,6 +57,26 @@ object results based on the [entity][entities] provided in the top-level [`from`
       }
     ]
 
+The same query can also be executed using [PQL][pql]:
+
+    curl -X GET http://localhost:8080/pdb/query/v4 --data-urlencode 'query=nodes { certname = "macbook-pro.local" }'
```

Comment:
This is the only place in the docs I have found that tell you how to execute a PQL query. Seems like we should at least link to it from the reference.